packages/next/src/build/webpack-config.ts

```diff
@@ -1284,6 +1286,17 @@ export default async function getBaseWebpackConfig(
             or: WEBPACK_LAYERS.GROUP.neutralTarget,
           },
         },
+        {
+          test: /[\\/].*?\.node$/,
+          loader: isNodeServer
+            ? 'next-server-binary-loader'
+            : 'next-error-browser-binary-loader',
```

Comment:
Couldn't this be in `handleExternals` instead of a custom loader?

Reply:
It won't be able to distinguish the `.node.js` and `.node`. 
For example a request in react-dom using `./server.node`, you need to apply a module resolution to the local relative import path, then you know if it's a `.node.js` file or just `.node`. 

It's causing extra effort and we didn't apply absolute resolution to the `.node`. And we also need to do it for browser/edge runtime.

Loader is easier to locate the actual `.node` file by regex, and we also know the absolute path which is easier to construct a require call

Reply:
Ah I see, makes sense